python/paddle/fluid/imperative/layers.py

```diff
@@ -53,6 +70,44 @@ def full_name(self):
         """
         return self._full_name
 
+    @property
+    def param_attr(self):
+        return self._helper.param_attr
+
+    @property
+    def bias_attr(self):
+        return self._helper.bias_attr
```

Comment:
this shouldn't live in Layer?

Reply:
this can totally hide helper, otherwise we will have `self._helper.bias_attr` usage in Layers building